optking/history.py

```diff
@@ -7,9 +8,11 @@
 
 from . import optparams as op
 from .exceptions import OptError
+from .molsys import Molsys
 from .linearAlgebra import abs_max, rms, sign_of_double
 from .printTools import print_array_string, print_mat_string
 from . import log_name
+from optking import molsys
```

Comment:
molsys is the module. Molsys is the class. instances of the class should be `lowercase` or `lowercase_with_underscore`.

I've never found a name I like for this our molecule. in 2017 I think it was `Molsys` see #15 I changed it in 2018 to `oMolsys`  to differentiate between the class and instances of the class. I've been slowly refactoring to `o_molsys` or `molsys` since then. Do you have a preferred name? `mol` is generally an external molecule in my mind.